ui/Model.elm

```diff
@@ -27,10 +27,21 @@ getBuildRecordList : Cmd Msg
 getBuildRecordList =
     client
         |> Kinto.getList recordResource
+        |> Kinto.limit 10
```

Comment:
should we put something like 100 here? It seems filtering 100 objects in memory (if we'll keep doing that?) shouldn't be too memory/cpu intensive